app/src/main/java/grinnell/appdev/edu/lyles/fragments/BeersAdapter.java

```diff
@@ -0,0 +1,253 @@
+package grinnell.appdev.edu.lyles.fragments;
+
+import android.animation.ObjectAnimator;
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.AsyncTask;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.util.List;
+
+import grinnell.appdev.edu.lyles.R;
+
+/**
+ * Created by nannan on 2017/2/12.
+ */
+
+// Create the basic adapter extending from RecyclerView.Adapter
+// Note that we specify the custom ViewHolder which gives us access to our views
+public class BeersAdapter extends RecyclerView.Adapter<BeersAdapter.ViewHolder> {
+
+    // Provide a direct reference to each of the views within a data item
+    // Used to cache the views within the item layout for fast access
+    public static class ViewHolder extends RecyclerView.ViewHolder {
+        // Your holder should contain a member variable
```

Comment:
This looks like a comment from a tutorial, so we can get rid of it (in the future, add a comment linking to the source)